### PR TITLE
Fix Sourceforge Links (#933)

### PR DESCRIPTION
**Pull request by RoiEXLab**
This adresses 99% of #933
When this PR is merged the remainng link needs to be updated - this is in order to prevent tests (if any) from failing due to a not available URL - Also this adds a version.properties File previously hosted on sourceforge - cleaned up and up to date (Newest version is 1.8.0.9 but whatever).
Also unused getters (and their according variables have been removed)

EDIT: Nevermind... Changed the Link now! Should work!

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/1000)

### CODE REVIEW

File: src/games/strategy/engine/framework/EngineVersionProperties.java [hunk 1]
```diff
@@ -20,39 +18,28 @@
...
 import games.strategy.util.Version;
 
 public class EngineVersionProperties {
   private final Version m_latestVersionOut;
-  private final Version m_showUpdatesFrom;
   private final String m_link;
   private final String m_linkAlt;
   private final String m_changelogLink;
-  private volatile boolean m_done = false;
   private static final String s_linkToTripleA = "http://triplea.sourceforge.net/latest/latest_version.properties";
```
Comment (RoiEXLab) [inline src/games/strategy/engine/framework/EngineVersionProperties.java]: This link needs to be changed later!

File: src/games/strategy/engine/framework/EngineVersionProperties.java [hunk 2]
```diff
@@ -71,79 +58,42 @@ private static EngineVersionProperties contactServerForEngineVersionProperties(f
...
-            ref.set(new EngineVersionProperties(engineversionPropsURL));
-            latch.countDown();
-          }
-        });
-        t1.start();
+      if (ref.get() != null) {
+        break;
       }
-    };
-    r.run();
```
Comment (RoiEXLab) [inline src/games/strategy/engine/framework/EngineVersionProperties.java]: It's stupid to create an anonymous Runnable class just to call the run method...

File: src/games/strategy/engine/framework/EngineVersionProperties.java [hunk 3]
```diff
@@ -71,79 +58,42 @@ private static EngineVersionProperties contactServerForEngineVersionProperties(f
...
-    method.setRequestHeader("User-Agent", "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.0)");
     try {
-      client.executeMethod(method);
-      final String propsString = method.getResponseBodyAsString();
-      props.load(new ByteArrayInputStream(propsString.getBytes()));
-    } catch (final Exception ioe) {
-      // do nothing, we return an empty properties
-    } finally {
-      method.releaseConnection();
+      props.load(new URL(s_linkToTripleA).openStream());
```
Comment (RoiEXLab) [inline src/games/strategy/engine/framework/EngineVersionProperties.java]: Too complicated (the previous stuff)... This works just fine

**Comment by RoiEXLab** [comment]
@DanVanAtta resolved conflicts...

File: latest_version.properties [hunk 4]
```diff
@@ -0,0 +1,4 @@
+LATEST=1.8.0.9
```
Comment (DanVanAtta) [inline latest_version.properties]: May as well be 1.9.0.0, 1.8.0.9 games will never read this file

**Pull request merged**